package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@babel/core": "7.2.2",
     "@svgr/webpack": "4.1.0",
+    "@types/hellojs": "^1.16.2",
```

Comment:
Why are we pulling in hellojs?  We should be using MSAL.js here.

Reply:
When pulling in MSAL.js, pull in an exact version. That will help us track changes to our dependencies. The caret before package version means npm will install the latest major change. For example if package version is ^1.16.2 in package.json and there's 1.17.0 availiable, during npm install version 1.17.0 of the package will be installed.

Use `npm i <package-name> --exact` to pull an exact version of a dependency.